### ISSUE TITLE
Defect: Lab 3 Exercise issues

### ISSUE THREAD

**Issue by Mster115** [all-rit/ALL#156]
# Defect: 
- Beginner Exercise: After repair, while the blue header says Accessible Exercise, the screen reader still says inaccessible exercise
- Advanced Exercise: Screen reader says 'cat' instead of 'image of cat' in the "Discover the Problem" page

# Acceptance Criteria:
- [x] Beginner Exercise screen reader says accessible exercise correctly
- [x] Advanced Exercise cat now says image of cat in discover page
- [x] Everything else should be unaffected
